Week2/Homework/Grade-calculator.js

```diff
@@ -0,0 +1,48 @@
+ 
+
+  let score=prompt("Enter your score"); // Manual input
```

Comment:
Just a FYI - it is not super encouraged to make use of `alert` or `prompt` APIs of browser as it is not the best of user experience.

Reply:
Replaced , thanks for letting me know .